frontend/integration-tests/views/service-catalog.view.ts

```diff
@@ -14,3 +14,4 @@ export const createBindingFormIsLoaded = () =>
     .wait(until.presenceOf($('.co-create-service-binding__name')), 10000)
     .then(() => browser.sleep(500));
 export const createButton = $('.co-m-btn-bar').element(by.buttonText('Create'));
+export const trySampleButton = $('.pf-c-button.pf-m-link.pf-m-inline');
```

Comment:
That's not exactly what I had in mind. It's still just a reference to a button. But it could be any button. The one above for example checks for the text.